client/src/actors.rs

```diff
@@ -1,15 +1,24 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod client;
-mod internal;
+mod registry;
+mod secure;
```

Comment:
The secure module contains the actor that is responsible for accessing the secure features inside stronghold, so I would argue the name make sense. The `SecureActor` renders the `ClientActor` and `InternalActor` obsolete, while providing the same level of isolation.

Reply:
Hmm, honestly it could go either way.  Since none of this is being exposed through the API it doesn't really matter except to people who are working on stronghold internally.